front/src/utils/__tests__/trainId.spec.ts

```diff
@@ -184,3 +174,11 @@ describe('getOccurrenceIndexFromOccurrenceId', () => {
     );
   });
 });
+
+describe('getPacedTrainIdFromOccurrenceId', () => {
```

Comment:
```suggestion
describe('getOccurrenceIndexFromOccurrenceId', () => {
```

Reply:
oh sorry i test the wrong function for the good expect my bad